vxfreeswitch/voice.py

```diff
@@ -392,12 +420,10 @@ def send_outbound_message(self, client, message):
 
         overrideURL = None
         client.set_input_type(None)
```

Comment:
I think we can delete the two lines above now since `overrideURL` and `input_type` are always set below?